packages/cli/src/commands/init-flow.ts

```diff
@@ -119,3 +112,7 @@ export const initFlow = async (
       .join("\n")
   );
 };
+
+export const convertToTitleCase = (str: string) => {
```

Comment:
we are using in builder import { titleCase } from "title-case";

Reply:
and we also have a humanizeString utility, but I guess you only need title-case

Reply:
But the package seems to change the `netlify-functions` to `
Netlify-Functions` but we are displaying like `Netlify Functions`. Is it fine ? I don't have much of a opinion on how to show. But the package claims to use english rules. So, sounds promising, changing it now 👍